common/djangoapps/util/views.py

```diff
@@ -84,6 +84,7 @@ def inner(request, *args, **kwargs):
                     # In debug mode let django process the 500 errors and display debug info for the developer
                     raise
                 else:
+                    log.exception("Error in django view.")
```

Comment:
It seems me general exception. We can get its detail a little more in  **e** i.e. `except Exception as e:`